Datascience_With_Python/Machine Learning/Algorithms/Logistic Regression/README.md

```diff
@@ -40,15 +40,26 @@ In Logistic regression, sigmoid function which is an 'S' shaped graph is used fo
 Hypothesis is taken as follows:
 
 Z = WX + B
+
 This Z is passed into the sigmoid function
 
 hΘ(x) = sigmoid (Z)
+
 where sigmoid function is given by:
 
 sigmoid(t)=1/(1+e^-t)
+
+<p align="center">
+  <img src="Sigmoid Curve.png" width="350" title="sigmoid curve">
+</p>
```

Comment:
I'm not able to view the images that have been added in the README file, @prathimacode-hub are you able to view the images that have been added?